AUTHORS.rst

```diff
@@ -0,0 +1,13 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* Sven Fillinger <sven.fillinger@qbic.uni-tuebingen.de>
+
+Contributors
+------------
+
+None yet. Why not be the first?
```

Comment:
https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners

This should either be CODEOWNERS or be removed.

Reply:
This is part of the QUBE template and needs to be adjusted there.